app/e2e/common/pages/playwright-landing-page.ts

```diff
@@ -0,0 +1,130 @@
+        this.UCRTecdLogo = page.locator('img[alt="ucr tecd logo"]');
+
+        this.loginFormModal = page.locator('[data-test-id="login-form"]');
+        this.registerFormModal = page.locator('[data-test-id="register-form"]');
+    }
+    // Dynamic Locators
+    async locateSideNavText(text: string) {
+        return this.page.locator(`div[role="button"]:has-text("${text}")`);
+    }
+    // Actions
+    async goto() {
+        await this.page.goto('/');
+    }
+
+    async clickOnLogin() {
+        await this.appBarLoginButton.click();
+    }
+
+    async clickOnRegister() {
+        await this.appBarRegisterButton.click();
+    }
+
+    async openSideNav() {
+        await this.appBarHamburgerButton.click();
+    }
+
+    async presEscape() {
+        this.page.keyboard.press('Escape');
+    }
+
+    async fillInFirstName(firstName: string) {
+        const firstNameInput = this.page.locator('text=First Name *First Name * >> input[type="text"]');
+        await expect(firstNameInput).toBeVisible();
+        await firstNameInput.click();
+        await firstNameInput.fill(firstName);
+    }
+
+    async fillInLastName(lastName: string) {
+        const lastNameInput = this.page.locator('text=Last Name *Last Name * >> input[type="text"]');
+        await expect(lastNameInput).toBeVisible();
+        await lastNameInput.click();
+        await lastNameInput.fill(lastName);
+    }
+
+    async fillInEmail(email: string) {
+        const emailInput = this.page.locator('input[type="email"]');
+        await expect(emailInput).toBeVisible();
+        await emailInput.fill(email);
+    }
+
+    async fillInPassword(password: string) {
+        const passwordInput = this.page.locator('input[type="password"]').first();
+        await expect(passwordInput).toBeVisible();
+        await passwordInput.fill(password);
+    }
+
+    async fillInConfirmPassword(password: string) {
+        const confirmPasswordInput = this.page.locator('input[type="password"]').nth(1);
+        await expect(confirmPasswordInput).toBeVisible();
+        await confirmPasswordInput.fill(password);
+    }
+
+    async submitLoginForm() {
+        const loginButton = this.page.locator('div[role="dialog"] button:has-text("Login")');
+        await expect(loginButton).toBeVisible();
+        await Promise.all([this.page.waitForNavigation(), loginButton.click()]);
+    }
+
+    async submitRegisterForm() {
+        const registerButton = this.page.locator('div[role="dialog"] button:has-text("Register")');
+        await expect(registerButton).toBeVisible();
+        await Promise.all([this.page.waitForNavigation(), registerButton.click()]);
+    }
+
+    async clickOnLargeRegisterButton() {
+        await expect(this.largeRegisterButton).toBeVisible();
+        await Promise.all([this.page.waitForNavigation(), this.largeRegisterButton.click()]);
+    }
+    // Route Checks
+    async amLoggedIn() {
+        await expect(this.page).toHaveURL('/organizations/me');
```

Comment:
Should this be something different? Like checking the cookie?

Reply:
We could do something like that I'm sure, even though it is a protected route I am not 100% sure we couldn't get a false positive so probably better to be safe than sorry. Also the route is subject to change to the cookie route is probably our best bet.